pathod/pathod.py

```diff
@@ -4,7 +4,8 @@
 import os
 import sys
 import threading
-import urllib
+
+from six.moves.urllib.parse import unquote
```

Comment:
@cortesi and I discussed yesterday that we do not want to import individual functions (as per https://google.github.io/styleguide/pyguide.html). Can we change this to `from six.moves import urllib`?